python/observation-publisher/conf/environment/example-override.yaml

```diff
@@ -0,0 +1,47 @@
+model:
+  id: "test-model"
+  # Supported model types:
+  #  - BINARY_CLASSIFICATION
+  #  - MULTICLASS_CLASSIFICATION
+  #  - REGRESSION
+  # - RANKING
+  type: "BINARY_CLASSIFICATION"
+  version: "0.1.0"
+  schema:
+    # Data types for features and prediction columns. Supported types are:
+    #  - INT64
+    #  - FLOAT64
+    #  - STRING
+    column_types:
+      accept: "BOOLEAN"
+      label: "STRING"
+      distance: "INT64"
+      transaction: "FLOAT64"
+    # Prediction result
+    prediction_score_column: "accept"
+    # Column name to be used for timestamp.
+    timestamp_column: "request_timestamp"
+    # Optional: User defined labels to group predictions
+    prediction_label_column: "label"
+    # Optional: Column name to be used for prediction id.
+    # If not provided, it's assumed to be prediction_id.
+    prediction_id_column: "prediction_id"
+observability_backend:
+  # Supported backend types:
+  #  - ARIZE
+  type: "ARIZE"
+  arize_config:
+    api_key: "SECRET_API_KEY"
+    space_key: "SECRET_SPACE_KEY"
+consumer:
```

Comment:
Nit: Instead of `consumer`, would `observations_source` be more meaningful? The "consumer" seems more suitable for Kafka but not as much for, say BQ source.